vip-api/src/main/java/fr/insalyon/creatis/vip/api/business/DataApiBusiness.java

```diff
@@ -531,6 +522,19 @@ private boolean baseDoesFileExist(String path) throws ApiException {
         }
     }
 
+    private boolean baseIsDirectory(String path) throws ApiException {
+        Optional<Type> type = baseGetPathInfo(path);
+        return type.isPresent() && type.get() == Data.Type.folder;
```

Comment:
I think i prefer using the `Data.Type` type for more clarity.
Also, use `equals` instead of `==`